documentation/alert/alert.html

```diff
@@ -0,0 +1,81 @@
+            </div>
+            <div class="info  alert">
+                <h3><img class="alert-icon" src="/documentation/alert/images/info.png" alt=""> Info!</h3>
+            </div>
+            <div class="light alert">
+                <h3><img class="alert-icon" src="/documentation/alert/images/light.jpeg" alt=""> Light!</h3>
+            </div>
+            <div class="dark alert">
+                <h3><img class="alert-icon" src="/documentation/alert/images/dark.png" alt=""> Black!</h3>
+            </div>
+            <div class="code-container">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522danger%2520alert%2522%253E%250A%2509%2509%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Fdanger.jpeg%2522%250A%2520%2509%2509%2509alt%253D%2522%2522%253E%2520Danger%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522warning%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Fwarning.jpeg%2522%250A%2520%2509%2509%2509alt%253D%2522%2522%253E%2520Warning%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522success%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Fsuccess.png%2522%250A%2520%2509%2509%2509alt%253D%2522%2522%253E%2520Success%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522info%2520%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Finfo.png%2522%250A%2520%2509%2509%2509alt%253D%2522%2522%253E%2520Info%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522light%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Flight.jpeg%2522%250A%2520%2509%2509%2509alt%253D%2522%2522%253E%2520Light%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522dark%2520alert%2522%253E%250A%2509%2509%253Ch3%253E%253Cimg%2520class%253D%2522alert-icon%2522%2520src%253D%2522%252Fdocumentation%252Falert%252Fimages%252Fdark.png%2522%2520%250A%2509%2509%2509alt%253D%2522%2522%253E%2520Black%21%253C%252Fh3%253E%250A%253C%252Fdiv%253E"
+                    style="width: 833px; height: 639px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you can move the styles to the common css file

Reply:
Ok bro.